test/boost/s3_test.cc

```diff
@@ -20,13 +20,13 @@
 
 /*
  * Tests below expect minio server to be running on localhost
- * with the bucket named "testbucket" created with unrestricted
- * anonymous read-write access
+ * with the bucket named env['MINIO_PUBLIC_BUCKET'] created with
+ * unrestricted anonymous read-write access
  */
 
 SEASTAR_THREAD_TEST_CASE(test_client_put_get_object) {
     const ipv4_addr s3_server(::getenv("MINIO_SERVER_ADDRESS"), 9000);
-    const sstring name(fmt::format("/testbucket/testobject-{}", ::getpid()));
+    const sstring name(fmt::format("/{}/testobject-{}", ::getenv("MINIO_PUBLIC_BUCKET"), ::getpid()));
```

Comment:
Let's call it SCYLLA_TEST_BUCKET_NAME, we shoudl reserve MINIO namespace to minio.

Reply:
(it's also not minio if you run it over s3).

Reply:
and also - default to testbucket so it's possible to run the test without failing over missing environment variables. But I see there's already getenv("MINIO") calls, we can fix all this later.

Reply:
Fixed in #13546